pkg/apis/pipeline/v1beta1/taskref_validation.go

```diff
@@ -35,6 +37,7 @@ func (ref *TaskRef) Validate(ctx context.Context) (errs *apis.FieldError) {
 	switch {
 	case ref.Resolver != "" || ref.Params != nil:
 		if ref.Resolver != "" {
+			errs = errs.Also(version.ValidateEnabledAPIFields(ctx, "resolver", config.BetaAPIFields).ViaField("resolver"))
```

Comment:
can you please add tests for this change?

Reply:
Thanks! Added and similarly for pipelineref test.